### ISSUE TITLE
theme.info file styling

### ISSUE THREAD

**Issue by alan-cole** [mastazi/VS-code-drupal#1]
Currently the listed filetype support does not include the .info files found in Drupal 7 themes. Would it be possible to add support for .info as well?

**Comment by mastazi** [comment]
thank you for spotting this! If I remember correctly, .info files use the .ini syntax, is that right? 
I'm at work now but will include this as soon as I have access to my home computer. 
Obviously, if you like, you can submit a pull request :-)

**Comment by mastazi** [comment]
Hi @alan-cole I have just published the new version 0.12 which takes care of highlighting .info files (which are used both in Drupal themes and Drupal modules), note that .info files use the .ini grammar so that's what I'm using, however it's still not perfect because array entries are not highlighted e.g. 

```
stylesheets[all][] = css/layout.css
stylesheets[all][] = css/style.css
```

I will research a bit and see if I can fix that.

**Comment by mastazi** [comment]
Closing for now, the issue with arrays has been submitted to the VS Code main project https://github.com/demyte/language-cshtml/issues/13
